admin/src/pages/UserSearch.vue

```diff
@@ -48,6 +57,11 @@ export default {
   },
 
   methods: {
+    unconfirmedRegisterMails() {
+      this.searchResult = this.searchResult.filter((result) => {
```

Comment:
Please do this in the backend. When we have pagination then everything breaks with this solution